src/components/search/search.njk

```diff
@@ -1,35 +1,35 @@
 <h6>Search big</h6>
 
-<div role="search">
-  <form class="usa-search usa-search-big">
-    <label class="usa-sr-only" for="search-field-big">Search big</label>
+<form class="usa-search usa-search-big">
+  <div role="search">
```

Comment:
would it make sense to include the `role` with the `form` element, remove the inner `div`, and consolidate the styles? Do we gain some important isolation/control by keeping them separate?

Reply:
No, it's for accessibility reasons, form has an inherent role. http://www.a11ymatters.com/pattern/accessible-search/

Reply:
Gotcha. Can/should any of the rules tied to `[role=search]` be applied to `usa-search` instead?

Reply:
No bc `usa-search` is the form element and `[role=search]` is the inner div and they have different style needs.